### PR TITLE
Updating docs for 0.8.0

### PR DESCRIPTION
**Pull request by alex-glv**
Add minimesosFile section
Add install / use sections

### CODE REVIEW

**Comment by sadovnikov** [comment]
Approved

**Comment by frankscholten** [comment]
LGTM

File: docs/index.md [hunk 1]
```diff
@@ -43,6 +43,78 @@ Once the installation has been successful, let's try running ```minimesos --help
 This should print the list of all possible commands and command line arguments.
 
 
+These are the options you might want to change to configure your cluster.
```
Comment (alex-glv) [inline docs/index.md]: Not sure why it was moved from here.
This was meant to be under the main configuration options

**Comment by alex-glv** [comment]
@sadovnikov Viktor, I think you have made some commits after the approval which went unnoticed. I think we should create separate pull requests, or at least not merge if the pr had been approved  before new commits appeared.

**Pull request merged**